packages/yourloops/lib/auth/hook.tsx

```diff
@@ -87,6 +97,35 @@ function AuthContextImpl(api: AuthAPI): AuthContext {
     return Promise.reject(new Error(t("not-logged-in")));
   };
 
+  /**
+   * Update hook users infos, perform zendesk & matomo login
+   * @param st sessionToken
+   * @param tt traceToken
+   * @param u user
```

Comment:
I know, I didn't have much imagination the day I wrote that.
Since sessionToken & others exists in a "upper" level, I didn't want to use them.
I will think to others name.